### PR TITLE
Community Article - PayPal Requiring Use of TLS 1.2

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-23c73aef48/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-993ed64f91/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-006f20f1ad/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-c59838a49c/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-253e6beea6/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-074d0b8375/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c7ed3f301d/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-0d519e44fe/how-to/).

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
@RackerKate On it.

**Comment by cyrichardson** [comment]
@RackerKate Suggestions here: 
[paypal-requiring-tls-12_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2890274/paypal-requiring-tls-12_cyrEdits.docx)

**Comment by cyrichardson** [comment]
@RackerKate I forgot to mark this... consider changing the title to use "requires" rather than "requiring".

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5f3ce3a25a/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-046a7cc2dd/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-e66905a883/how-to/). :bow:

File: content/cloud-servers/paypal-requires-tls-12.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+### <a id="rhel_and_centos6">RHEL and CentOS 6</a>
+
+RHEL 6.8 enables TLS 1.2 use by default. In both RHEL and CentOS, customers
+who rely on Rackspace for automatic patching automatically update in their
+normal patching cycle.
+
+### <a id="rhel_and_centos5">RHEL and CentOS 5</a>
+
+RHEL and CentOS 5 do not support TLS 1.2. As a result, your websites and
+applications will no longer be able to take payments by using the PayPal
```
Comment (catlook) [inline content/cloud-servers/paypal-requires-tls-12.md]: applications can no longer take payments ...

File: content/cloud-servers/paypal-requires-tls-12.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+service because the server is no longer able to talk to the PayPal endpoint.
```
Comment (catlook) [inline content/cloud-servers/paypal-requires-tls-12.md]: ... server can no longer talk to ...

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b8ab1b4b39/how-to/)

**Pull request merged**